ngrok_test.go

```diff
@@ -0,0 +1,75 @@
+package ngroklistener
+
+import (
+	"time"
+	// "reflect"
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	// "github.com/caddyserver/caddy/v2"
```

Comment:
remove the commented lines here and I'll immediately merge this PR 🙂

Reply:
Done! Thank you!